src/gui/plugins/insert_model/InsertModel.qml

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.9
+import QtQuick.Window 2.2
+import QtQuick.Controls 2.1
+import QtQuick.Controls.Material 2.2
+import QtQuick.Controls.Material.impl 2.2
+import QtQuick.Layouts 1.3
+import QtQuick.Controls.Styles 1.4
+
+Rectangle {
```

Comment:
We've already talked about this in person, but I'll just leave some UI notes here so we don't forget:

- [ ] All text should be the same size
- [ ] Use [Text.ElideRight](https://doc.qt.io/qt-5/qml-qtquick-text.html#elide-prop) for names
- [ ] Support light / dark modes
- [ ] Use [Material.elevation](https://doc.qt.io/qt-5/qtquickcontrols2-material.html#material-elevation-attached-prop) to give each item a "card" look
- [ ] Add more padding to the edges of the plugin, and between the items
- [ ] Change the cursor to [Qt.WaitCursor](https://doc.qt.io/qt-5/qml-qtquick-mousearea.html#cursorShape-prop) while the preview is being spawned

Reply:
First four points are resolved in 47f214cbf4408a95015531257055d17db667184f